src/components/ToolbarButton.tsx

```diff
@@ -54,11 +54,19 @@ const ToolbarButtonComponent: FC<DraggableToolbarButtonProps> = ({
     throw new Error('The svg property is required to render a shortcut in the Toolbar. ' + shortcutId)
   }
 
-  const commandState = commandStateStore.useSelector(state => state[shortcutId])
   const isDraggingAny = useSelector(state => !!state.dragShortcut)
   const dragShortcutZone = useSelector(state => state.dragShortcutZone)
-  const isButtonActive =
-    useSelector(state => (customize ? selected : !isActive || isActive(() => state))) || (!customize && commandState)
+
+  const isCommandActive = commandStateStore.useSelector(state => state[shortcutId])
+  const isButtonActive = useSelector(state => {
+    if (isCommandActive) {
+      return true
+    }
+    if (customize) {
+      return selected
+    }
+    return !isActive || isActive(() => state)
```

Comment:
Prefer ternary expressions for short logic, and a single return statement at the end of the function.

Reply:
Switched to a single return statement with a couple of intermediate variables: 3ed481701f4d26d84bc18ff807a7bf950c52205b

Then, further refactored per the other comment.